### PR TITLE
POM view of own caseload

### PR DESCRIPTION
**Pull request by KPobeeNorris**
This feature allows signed in POMs to view all their allocated cases, and specifically those newly allocated cases from the past 7 days.

### CODE REVIEW

File: app/controllers/dashboard_controller.rb [hunk 1]
```diff
@@ -1,5 +1,9 @@
 class DashboardController < ApplicationController
   before_action :authenticate_user
 
-  def index; end
+  def index
+    user = Nomis::Elite2::Api.fetch_nomis_user_details(current_user).data
```
Comment (rossjones) [inline app/controllers/dashboard_controller.rb]: Should this be in a Service?

File: app/controllers/poms_controller.rb [hunk 2]
```diff
@@ -19,6 +22,22 @@ def show
 
   def edit; end
 
+  def my_caseload
+    user = Nomis::Elite2::Api.fetch_nomis_user_details(current_user).data
```
Comment (rossjones) [inline app/controllers/poms_controller.rb]: Probably in a service like the others, and then that call wrapped in a method called user?

File: app/controllers/dashboard_controller.rb (hunk 1 above)
Comment (TobyRet) [reply app/controllers/dashboard_controller.rb]: Yep agree!

File: app/controllers/poms_controller.rb [hunk 3]
```diff
@@ -19,10 +22,27 @@ def show
...
+
 private
 
   def pom
     poms_list = PrisonOffenderManagerService.get_poms(caseload)
     @pom = poms_list.select { |p| p.staff_id.to_i == params['id'].to_i }.first
   end
+
+  def signed_in_pom_details
+    user = Nomis::Elite2::Api.fetch_nomis_user_details(current_user).data
```
Comment (TobyRet) [inline app/controllers/poms_controller.rb]: Same here

**Pull request merged**